### DISCUSSION TITLE
falha no build - Fundamentos C# (Editor de texto)

### DISCUSSION THREAD

**Discussion by leticialauriano** [balta-io/forum#361] [Tira Dúvidas] [answered]
Fui fazer um dotnet run após fazer a parte do static void Salvar(string texto), e começou a dar este erro.
Antes estava rodando normalmente. 
Segue imagens do código completo e do terminal com o erro. 

![1](https://github.com/balta-io/forum/assets/114084854/d9ef86bc-a670-42f1-96f4-027d58710561)
![2](https://github.com/balta-io/forum/assets/114084854/50bb9d75-622e-4d8d-a6e3-8f49899c70b5)

Obrigada!

**Comment by leticialauriano** [comment, marked as answer]
Resolvido, encontrei o erro.
Estava com mais instâncias abertas do powershell e estava dando erro.

Obrigada! 👏

**Reply by andrebaltieri** [reply]
Show @leticialauriano!!!!
